Provider/VersionProvider.php

```diff
@@ -37,9 +37,9 @@ public function isSupported()
      */
     public function getVersion()
     {
-        $result = file_get_contents($this->path . DIRECTORY_SEPARATOR . 'VERSION');
+        $result = fgets(fopen($this->path . DIRECTORY_SEPARATOR . 'VERSION', 'rb'));
 
-        return $result;
+        return trim($result);
```

Comment:
I would suggest to use `rtrim()` here and checking for `''` instead of `false` in `canGetVersion()`